packages/apps-engine/src/server/AppStatusCache.ts

```diff
@@ -0,0 +1,40 @@
+import type { AppStatus } from '../definition/AppStatus';
+
+export class AppStatusCache {
+    private readonly cache: Record<string, { status: AppStatus, expiresAt: Date }> = {};
+    private readonly CACHE_TTL = 1000 * 60;
+
+    // We clean up the cache every 5 minutes
+    private readonly CLEANUP_INTERVAL = 1000 * 60 * 5;
```

Comment:
give them environment variables so we can configure from outside?

Reply:
I feel like we've been adding too many environment variables to configure small parts of the apps-engine, it's getting a bit out of hand.

I was thinking about something similar to the ts-node env var that gets a JSON string as value, like `TS_NODE_COMPILER_OPTIONS='{\"module\": \"commonjs\"}'`. What do you think? @debdutdeb

Reply:
Oh yeah that sounds better with that context. :+1: